packages/react-components/src/organisms/ResearchOutputIdentifier.tsx

```diff
@@ -65,6 +68,42 @@ const identifierMap = {
   },
 } as const;
 
+const getIdentifierInfoMessage = (
+  identifiers: IdentifierType,
+): Array<ReactElement> =>
+  identifiers
+    .filter(({ value }) => value !== ResearchOutputIdentifierType.None)
+    .map(({ value }) => {
+      if (value === ResearchOutputIdentifierType.AccessionNumber) {
+        return (
+          <InfoParagraph
+            key={value}
+            boldText="Accesion Number: "
+            bodyText={
+              identifierMap[ResearchOutputIdentifierType.AccessionNumber]
+                .helpText
+            }
+          />
```

Comment:
I'm thinking we can have this just once and render it based on the ResearchOutputIdentifierType

Reply:
```
 boldText="Accesion Number: "
```

That could change to : 
```
bodyText={
              identifierMap[ResearchOutputIdentifierType.AccessionNumber]
                .name
            }
```

Reply:
If we add `Accesion Number` to the identifier map as Name. The ' :' should belong in the component?